app/(main)/idea/components/IdeaInputCard.tsx

```
@@ -872,6 +967,59 @@ export default function IdeaInputCard({
       </div>
         );
       })()}
+      
+      {/* Public Repository Dialog */}
+      <Dialog open={publicRepoDialogOpen} onOpenChange={setPublicRepoDialogOpen}>
+        <DialogContent className="sm:max-w-[425px]">
+          <DialogHeader>
+            <DialogTitle>Enter Public Repository</DialogTitle>
+            <DialogDescription>
+              Enter a public GitHub repository name in the format "owner/repo" (e.g., facebook/react)
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)